charts/couchdb/templates/statefulset.yaml

```diff
@@ -63,20 +63,6 @@ spec:
 {{- end }}
             - name: ERL_FLAGS
               value: "{{ range $k, $v := .Values.erlangFlags }} -{{ $k }} {{ $v }} {{ end }}"
```

Comment:
Thought -- Still wondering if we should keep-it and make timeout longer... (how much longer?)

Reply:
Let's omit them for now to avoid issues with slower systems. When we fix the pod bring-up dependencies we can re-introduce the probes.